InfrastructureManager/agents/factory.py

```diff
@@ -1,7 +1,9 @@
+from agents.azure_agent import AzureAgent
```

Comment:
Is this the same code as the tools have? Could you just do `from appscale.tools.agents.azure_agent import AzureAgent`, or is there a difference between the two agents?